docker/env/version

```diff
@@ -1 +1 @@
-1.39-azure-storage
+1.39-remove-libcloud
```

Comment:
Please, bump also integer part. Looking at the image tags not knowing context/history it is unclear which one is newer.

Reply:
changed the version number

I was neglecting it, since I have two commit creating new docker image in the same PR.